frontend-react/src/senders/hooks/UseSenderSchemaOptions/index.ts

```diff
@@ -0,0 +1,70 @@
+import { useSenderResource } from "../../../hooks/UseSenderResource";
+import { FileType } from "../../../hooks/UseFileHandler";
+
+export enum StandardSchema {
+    CSV = "upload-covid-19",
+    CSV_OTC = "csv-otc-covid-19",
+    HL7 = "standard/standard-covid-19",
```

Comment:
Just a reminder to change this to 'standard-covid-19'

Reply:
Ah, this should actually be `standard/standard-covid-19` -- it's based on the filepaths in [metadata/schemas](https://github.com/CDCgov/prime-reportstream/tree/master/prime-router/metadata/schemas), just normalized/lowercased.  So `WATERS/waters-covid-19` is `waters/waters-covid-19`, `Experity/experity-covid-19`  is `experity/experity-covid-19`, `standard/standard-covid-19` is `standard/standard-covid-19`, et cetera.

You can check out `schemaStore.keys` in [Metadata.kt](https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/src/main/kotlin/Metadata.kt#L201-L203) to check up on the normalized names:

<img width="246" alt="Screenshot 2023-02-08 at 20 53 34" src="https://user-images.githubusercontent.com/2421042/217697242-d2bdf830-5216-4814-8bf3-97b0916f1b19.png">